plugin/trino-mongodb/src/test/java/io/trino/plugin/mongodb/TestMongoConnectorTest.java

```diff
@@ -902,6 +902,27 @@ protected Optional<SetColumnTypeSetup> filterSetColumnTypesDataProvider(SetColum
         return Optional.of(setup);
     }
 
+    @Override
+    public void testDropFieldFromRowType()
+    {
+        assertThatThrownBy(super::testDropFieldFromRowType)
+                .hasMessageContaining("This connector does not support dropping fields from row types");
```

Comment:
This shouldn't require an override.
the test class should declare testing connector behavior that it doesn't support that